.github/workflows/tests.yml

```
@@ -0,0 +1,96 @@
+name: Tests
+
+permissions: {}
+
+on:
+  push:
+    branches: ["**"]
```

need to swap this before merge

```suggestion
    branches: [ master ]
```